File: Programmers/코딩테스트_입문/GangHyeok/Day06/짝수 홀수 개수.js
function solution(num_list) {
  var answer = [0, 0];
  
  
  for(i of num_list){
      if(i%2 == 0){
          answer[0]++;
      }
      else{
          answer[1]++;
      }
  }
  
  return answer;
}

File: Programmers/코딩테스트_입문/GangHyeok/Day06/짝수 홀수 개수.java
class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = {};
        int odd=0, even=0;
        
        for(int i=0;i<num_list.length;i++){
            if(num_list[i]%2 == 0){
                even++;
            }
            else{
                odd++;
            }
        }
        
        answer = new int[2];
        answer[0] = even;
        answer[1] = odd;
        
        return answer;
    }
}